python/plugins/processing/algs/qgis/Rasterize.py

```diff
@@ -57,6 +57,7 @@
 
 
 class RasterizeAlgorithm(QgisAlgorithm):
+
```

Comment:
The spaces were not added manually, those were added automatically through pre commit hook's autopep. I'm running on Ubuntu's upcoming 17.10 version, seems the version of autopep offered through this version's repository is recent enough to have the above mentioned PEP formatting included.